packages/plugin-mcp/src/utils.ts

```
@@ -0,0 +1,68 @@
+import { StdioClientTransport } from "@modelcontextprotocol/sdk/client/stdio";
+import { StreamableHTTPClientTransport } from "@modelcontextprotocol/sdk/client/streamableHttp";
+
+import { ServerConfig, StdioConfig, StreamableHttpConfig } from "./types";
+
+export function clientCommand(
+  command: string | undefined,
+  serverScriptPath: string | undefined
+) {
+  let result;
+
+  if (command) {
+    result = command;
+  } else if (serverScriptPath) {
+    const isPython = serverScriptPath.endsWith(".py");
+    result = isPython
+      ? process.platform === "win32"
+        ? "python"
+        : "python3"
+      : process.execPath;
+  } else {
+    throw new Error(
+      "MCP config needs either {command,args} or serverScriptPath"
+    );
+  }
+
+  return result;
```

we can make `result` a `const` inside the scope of the conditional statements without mutable vars:

```ts
if (command) return command;

if (serverScriptPath) {
   const isPython = serverScriptPath.endsWith(".py");
   const command = isPython ? process.platform === "win32" ? "python" : "python3" : process.execPath;
   return command;
}

throw new Error("MCP config needs either {command,args} or serverScriptPath");

---

I prefer 1 entry 1 exit. ie no multiple returns in a function. 

if this is your preference then yes i can make it that

---

gotcha, yeah my preference is fail fast principle

---

Done.